runtime/Stdlib_ArrayBuffer.resi

```diff
@@ -0,0 +1,68 @@
+/***
+Functions for interacting with JavaScript ArrayBuffer.
+See: [`ArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer).
+*/
+
+/**
+Type representing an ArrayBuffer object used to represent a generic raw binary data buffer.
+*/
+@notUndefined
+type t
+
+/**
+`make(length)` creates a new ArrayBuffer with the specified length in bytes.
+See [`ArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/ArrayBuffer) on MDN.
+
+## Examples
+
+```rescript
+let buffer = ArrayBuffer.make(8)
+ArrayBuffer.byteLength(buffer)->assertEqual(8)
+```
+
+## Exceptions
+
+- `RangeError`: If `length` is larger than `Number.MAX_SAFE_INTEGER` or negative.
+*/
+@new external make: int => t = "ArrayBuffer"
+
+/**
+`byteLength(arrayBuffer)` returns the size, in bytes, of the ArrayBuffer.
+See [`ArrayBuffer.byteLength`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/byteLength) on MDN.
+
+## Examples
+
+```rescript
+let buffer = ArrayBuffer.make(16)
+ArrayBuffer.byteLength(buffer)->assertEqual(16)
```

Comment:
Sorry to nitpick, but I got the same feedback some time ago.
I believe we want to use `==` where possible instead of `->assertEqual`.